From 7f24e190bde4ee120e284cbdb59eed36ef8818f8 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Thu, 10 Sep 2020 15:44:09 +0800
Subject: [PATCH] CIM_Framework/CMakeLists.txt: turn OFF BUILD_LIBCIM

BUILD_LIBCIM forces a check for libcmpisfcc in the top level makefile
for openwsman while the cim plugin itself isn't built if DISABLE_PLUGINS
is turned ON. This leads to an error when that library isn't present
even though we're not building that part of the code at all.

Turn OFF BUILD_LIBCIM in addition to DISABLE_PLUGINS.

Upstream-Status: Submitted [https://github.com/intel/lms/pull/4]

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 CIM_Framework/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CIM_Framework/CMakeLists.txt b/CIM_Framework/CMakeLists.txt
index 9d287f9..60219b3 100644
--- a/CIM_Framework/CMakeLists.txt
+++ b/CIM_Framework/CMakeLists.txt
@@ -53,6 +53,7 @@ externalproject_add (openwsman
     -DUSE_PAM=OFF
     -DENABLE_EVENTING_SUPPORT=OFF
     -DBUILD_SHARED_LIBS=OFF
+    -DBUILD_LIBCIM=OFF
     ${WSMAN_STATIC_RUNTIME}
     ${WSMAN_ARCH}
     ${USE_TOOLCHAIN}
-- 
2.26.2

